src/main_gui.cpp

```diff
@@ -362,6 +363,10 @@ struct MainWindow : Window
 				UpdateAllVirtCoords();
 				break;
 
+			case GHK_FORCE_DESYNC: // Force a network desync by taking an unsactioned random number
```

Comment:
unsanctioned?

Reply:
Uh maybe a better word exists. A random number not needed by the game logic?